.github/workflows/ci.yml

```diff
@@ -42,3 +49,13 @@ jobs:
       - uses: fastify/github-action-merge-dependabot@v2.2.0
         with:
           github-token: ${{ secrets.GITHUB_TOKEN }}
+
+  coverage:
+    needs: test
+    runs-on: ubuntu-latest
+    steps:
+      - name: Coveralls Finished
+        uses: coverallsapp/github-action@1.1.3
+        with:
+          github-token: ${{ secrets.GITHUB_TOKEN }}
+          parallel-finished: true
```

Comment:
You can remove coveralls. It's 100% and we don't need to track it.

Reply:
done